iina/Info.plist

```diff
@@ -1334,6 +1334,40 @@
 				</array>
 			</dict>
 		</dict>
+    <dict>
+      <key>UTTypeConformsTo</key>
+      <array>
+        <string>public.audio</string>
+      </array>
+      <key>UTTypeDescription</key>
+      <string>Direct Stream Digital DSF File</string>
+      <key>UTTypeIdentifier</key>
+      <string>io.iina.dsf</string>
```

Comment:
Many of the current entries are incorrect. Per Apple you must not use your own UTI for someone else's file type. We need to try and find an official UTI for this file type.